### DISCUSSION TITLE
Cm_Redis_Session Update

### DISCUSSION THREAD

**Discussion by rafaelpatro** [OpenMage/magento-lts#2113] [Questions & Answers]
Any reason why [Cm_Redis_Session](https://github.com/colinmollenhour/Cm_RedisSession) is not updated here in this project?
I'm using latest release _based on abstracted handler_ for sessions since April and have no issues.

_P.s. I suppose it's the compiler. If so I'd like to suggest deactivating the compiler forever. Redis Session benefits overlaps compiler benefits._

**Comment by colinmollenhour** [comment]
No reason not to, I suppose most people using this repo are also using the latest version anyway.

**Comment by seansan** [comment]
+1

**Comment by seansan** [comment]
We use the solution

where community is set to false and local is included like so


![image](https://user-images.githubusercontent.com/652395/45748022-d3dbf200-bc07-11e8-858b-14936e1bad4f.png)

**Comment by luigifab** [comment]
Some news about the update? 😺

**Comment by grgfo** [comment]
Hello

We have some usleep on your website with redis_session. I think it's fixed on the last Redis Session module so it would be cool if it's included in openmage instead of an older version.
